src/styles/app/18-filetree-body.css

```
@@ -242,6 +370,33 @@
+}
+
+.fileTreeFilePreview {
+	display: -webkit-box;
+	-webkit-line-clamp: 1;
+	-webkit-box-orient: vertical;
+	overflow: hidden;
+	color: color-mix(in srgb, var(--text-tertiary) 82%, transparent);
+	font-size: 11px;
+	line-height: 1.2;
+	white-space: normal;
+	word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)